neural_compressor/compression/callbacks.py

```diff
@@ -257,45 +273,48 @@ def pre_process(self):
             self.remove_hook("on_train_begin", self.adaptor._pre_hook_for_qat)
             self.remove_hook("on_train_end", self.adaptor._post_hook_for_qat)
 
-        strategy = self.cfg.tuning.strategy.name.lower()
+        strategy = self.cfg.quantization.tuning_criterion.strategy.lower()
         if self.cfg.quantization.quant_level == 0:
             strategy = "conservative"
             logger.info(f"On the premise that the accuracy meets the conditions, improve the performance.")
 
         if strategy == "mse_v2":
-            if not (self.cfg.model.framework.startswith("tensorflow") or self.cfg.model.framework == 'pytorch_fx'):
+            if not (self.cfg.quantization.framework.startswith("tensorflow") \
+                    or self.cfg.quantization.framework == 'pytorch_fx'): # pragma: no cover
                 strategy = "basic"
-                logger.warning(f"MSE_v2 does not support {self.cfg.model.framework} now, use basic instead.")
+                logger.warning(f"MSE_v2 does not support \
+                               {self.cfg.quantization.framework} now, use basic instead.")
                 logger.warning("Only tensorflow, pytorch_fx is supported by MSE_v2 currently.")
         assert strategy in STRATEGIES, "Tuning strategy {} is NOT supported".format(strategy)
 
         _resume = None
         # check if interrupted tuning procedure exists. if yes, it will resume the
         # whole auto tune process.
-        self.resume_file = os.path.abspath(os.path.expanduser(self.cfg.tuning.workspace.resume)) \
-                           if self.cfg.tuning.workspace and self.cfg.tuning.workspace.resume else None
+        self.resume_file = os.path.abspath(os.path.expanduser(self.cfg.options.resume_from)) \
+                           if self.cfg.options.workspace and self.cfg.options.resume_from else None
```

Comment:
self.cfg.options are deprecated one. we are using func to set those global variables. pls confirm if this right implementation

Reply:
done